src/main/java/de/thomas_oster/liblasercut/drivers/GenericGcodeDriver.java

```diff
@@ -1057,6 +1073,28 @@ public void setApiKey(String apiKey)
     this.apiKey = apiKey;
   }
   
+  private int gCodeDigits;
```

Comment:
is the default value correct? what happens when importing old settings?

Reply:
I have not set default values and I have not tested importing existing setting files. When I fired up the new build they initialized with 0.

Reply:
I added default values and creating an new machine shows the appropriate values now. When importing an old configuration, these values will initialize with 0 and have to be set manually again. When importing settings that where exported AFTER this change, these values initialize with the values saved to the export as expected.